File: Conway's_GoF/Node.java
/*
Klassen Node.

Representerar ein datamaskin i ei rekneklynge.

Tek inn talet paa prosessorar og minnestorleik i GB som parametarar,
og kan returnera desse verdiane.
*/

public class Node
{
  //Oppretter instansvariablane:
  //Tal paa prosessorar
  private int antProsessorer;
  //Tal paa minne i GB
  private int minneGB;

  //Konstruktoeren, tek inn prosessortal og minnestorleik,
  //sett innverdiane til instansvariablane
  public Node(int minneGBInn, int antProsessorerInn)
  {
    antProsessorer = antProsessorerInn;
    minneGB = minneGBInn;
  }

  //Returnerar talet paa prosessorar
  public int lesAntProsessorer()
  {
    return antProsessorer;
  }

  //Returnerar talet paa minne i GB
  public int lesMinneGB()
  {
    return minneGB;
  }
}

File: java/Oblig1/python/node.py
"""
Held klassen Node.

Representerar ein datamaskin i ei rekneklynge.

Tek inn talet paa prosessorar og minnestorleik i GB som parametrar,
og kan returnera desse verdiane.
"""

class Node:
    #Konstruktoeren, tek inn prosessortal og minnestorleik som parametrar
    def __init__(self, antProsessorer, minneGB):
        #sett talet paa prosessorar
        self._antProsessorer = antProsessorer
        #sett minnestorleiken
        self._minneGB = minneGB


    #Returnerar talet paa prosessorar
    def antProsessorer(self):
        return self._antProsessorer

    #Returnerar minnestorleiken
    def minneIGB(self):
        return self._minneGB
